### DISCUSSION TITLE
throwsAsync() not working

### DISCUSSION THREAD

**Discussion by taitruong** [avajs/ava#3143] [Q&A]
Here is my code snippet:

```typescript
  // test max mint exceeded
  t.throwsAsync(() =>
    myFunReturningErrorPromise()
  );
```
So my promises gets rejected. I was expecting that `throwsAsync()` would pass my rejected promise, but instead I get an `Rejected promise returned by test.` message and my test fails/stops.

**Comment by tmillr** [comment]
@taitruong Did you make sure to await the `t.throwsAsync` call?

For example:

```ts
test("my test", async (t) => {
  await t.throwsAsync(
    myFunReturningErrorPromise()
  );
});
```

**Comment by taitruong** [comment]
Thx, it works as expected!
